firebase-installations/src/main/java/com/google/firebase/installations/FirebaseInstallations.java

```diff
@@ -409,7 +419,7 @@ private Void deleteFirebaseInstallationId() throws FirebaseInstallationsExceptio
 
     PersistedFidEntry persistedFidEntry = persistedFid.readPersistedFidEntryValue();
 
-    if (isPersistedFidRegistered(persistedFidEntry)) {
+    if (persistedFid.isFidRegistered(persistedFidEntry)) {
```

Comment:
this isn't right, is it? if `persistedFidEntry` is null then the `isFidRegistered` method will return true and then we'll call `persistedFidEntry.getFirebaseInstallationId()` right below which will NPE

Reply:
if ```persistedFidEntry``` is null then the ```isFidRegistered``` method will return false.

Reply:
Changes this.